templates/nextroute/main.go

```diff
@@ -21,9 +21,10 @@ func main() {
 }
 
 type options struct {
-	Model  factory.Options                `json:"model,omitempty"`
-	Solve  nextroute.ParallelSolveOptions `json:"solve,omitempty"`
-	Format nextroute.FormatOptions        `json:"format,omitempty"`
+	Model     factory.Options                `json:"model,omitempty"`
+	Solve     nextroute.ParallelSolveOptions `json:"solve,omitempty"`
+	Format    nextroute.FormatOptions        `json:"format,omitempty"`
+	NextCheck nextroute.NextCheckOptions     `json:"next_check,omitempty"`
```

Comment:
```suggestion
	Check check.Options     `json:"check,omitempty"`
```

If we organize the checker things into its own package, you avoid the next-next stuttering here. _Especially_ for options I find `check.Options` to be the options for the check.... Doesn't get simpler than that.